docs/one-click-integrations/bitbucket-integration.md

```diff
@@ -70,6 +75,14 @@ The Pulse Bitbucket integration detects incidents automatically using [pull requ
 -   The incident creation date is the timestamp when the reverted pull request was initially merged. If Pulse can't get the reverted pull request number from the branch name, the incident creation date is the timestamp of the first commit to the incident pull request.
 -   Pulse associates incidents to the system matching the repository name.
 
+### Use pull request naming convention (based on default branch) {: id="bb-incident-naming"}
+
+-   Pulse executes the same incident detection mechanism based on [pull request reverts](#bb-incident-pr-revert) using the naming convention you define instead of the default branch prefix `revert-pr-`.
+
+-   You can define the prefix for the branch name or the pull request title to match the naming convention on your workflow.
```

Comment:
See the [matching comment](https://github.com/codacy/pulse-user-docs/pull/198/files#r1116712058) for GitHub.

Reply:
```suggestion
-   You can define the prefix for the branch name or the pull request title to match the naming convention of your workflow.
```